addons/website_slides/static/src/js/slides_course_fullscreen_player.js

```diff
@@ -5,6 +5,7 @@ odoo.define('website_slides.fullscreen', function (require) {
     var sAnimations = require('website.content.snippets.animation');
     var Widget = require('web.Widget');
     var core = require('web.core');
+    var config = require('web.config');
```

Comment:
Commit message convention:
- the title must start with a conjugated verb. So 'stop showing sidebar on mobille page load'  instead of 'mobile default, hide sidebar' something like that (same comment for all commits)
- as your PR contains several commits, robodoo won't add PR references automatically, so add them by hand. For each commit, at the end of the commit message: "Part of https://github.com/odoo/odoo/pull/XXXXX"
- The user will have **to** click :wink: